### PR TITLE
Several GraphQL improvements.

### PR DESCRIPTION
**Pull request by fragalcer**
Fixes issues: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

**Comment by LegoStormtroopr** [comment]
I think we need these for Aristotle.cloud GraphQL bits

**Comment by fragalcer** [comment]
> I think we need these for Aristotle.cloud GraphQL bits

I checked already. We are not using those settings anymore.

File: python/aristotle-mdr-graphql/aristotle_mdr_graphql/types.py [hunk 1]
```diff
@@ -90,26 +87,25 @@ def __init_subclass_with_meta__(cls, *args, **kwargs):
...
-
-    def resolve_slots_as_object(self, info, **kwargs):
-        out = {}
-        for slot in slots_models.Slot.objects.get_item_allowed(self.item, info.context.user):
-            out[slot.name] = {
-                "type": slot.type,
-                "value": slot.value
-            }
-        return out
+    # TODO: REMOVE THESE METHODS BECAUSE THEY ARE NOT BEING USED ANYWHERE.
```
Comment (LegoStormtroopr) [inline python/aristotle-mdr-graphql/aristotle_mdr_graphql/types.py]: May need to keep. Double check

File: python/aristotle-mdr-graphql/aristotle_mdr_graphql/types.py [hunk 2]
```diff
@@ -1,15 +1,11 @@
-import graphene
 import logging
-
+from graphene import relay, String as GrapheneString
```
Comment (LegoStormtroopr) [inline python/aristotle-mdr-graphql/aristotle_mdr_graphql/types.py]: Can we change these back?

**Pull request merged**